mailbox/event/json/pom.xml

```diff
@@ -90,7 +85,7 @@
         <dependency>
           <groupId>org.scala-lang.modules</groupId>
           <artifactId>scala-java8-compat_${scala.base}</artifactId>
-          <version>0.9.0</version>
+          <version>${scala-java8_compat.version}</version>
```

Comment:
don't use properties for that, declare the dependencies into dependencyManagement of root pom.xml instead